### ISSUE TITLE
Seeding Report: Test Area Filter

### ISSUE THREAD

**Issue by braughtg** [DickinsonCollege/FD2School-FarmData2-F23#22]
The tests should generate seeding reports as appropriate to check that:
  - [ ] - when “All” is selected in the Area filter, the table will contain seeding logs for multiple different areas.
  - [ ] - when a specific area is selected in the Area filter, the table will contain only seeding logs for the selected area.
  - [ ] - the dropdown for the Area filter contains only areas for which there are seeding logs in the date range used to generate the report.

Some additional notes relevant to this issue:
  - The `.spec.js` file containing your test should be stored in an appropriate location and have a short but descriptive name.  Use the locations and an naming from the "Good First issues" as examples.
  - The `.spec.js` file should include a comment at the top that describes what the file as a whole is testing.
  - The message for the `describe` should describe in a short phrase what the file is testing.
  - After logging in and visiting the desired page the `beforeEach` method should call `cy.waitForPage()`.  This will ensure that the page is fully loaded (e.g. that all the `Maps` used by the page are loaded) before performing any tests.
  - It is not necessary to include a separate `it` for each of the things to be tested.
    - You should decide how to divide the things being tested into `it`s so that each `it` tests a cohesive set of things.
    - The message for each `it` should describe in a short phrase what the `it` is testing.
  - **The `.spec.js` files in the `farmdata2/farmdata2_modules/fd2_example/` sub-tabs (e.g. `ui`, `api`) may provide some helpful examples.**
  - These tests should utilize logs that are in the sample database.  Information about the data contained in the sample database can be found in the "The Sample Database" section of the `docker/sampleDB/README.md` file.

**Comment by Zivanovic8** [comment]
Group 1 will be working on this issue. (Aleksandar, Aidan, Fofang)

**Comment by anemervontage** [comment]
For the first goal, I wonder if it's possible to check for all the areas in our report data without having to manually make a cy.get statement for each one we see. Or is it better to simply check a few and move on? Basically, do you think there's a way to iterate over the list of areas we can see on the page and check to see if it matches the API data that is fetched by the page in Cypress? and if there was, would that be overly convoluted?

**Comment by Zivanovic8** [comment]
Yes, it is possible to dynamically check all the areas in your report data without manually creating a separate cy.get statement for each one. You can achieve this by leveraging JavaScript's array iteration methods and Cypress commands.

**Comment by anemervontage** [comment]
I see, that makes sense, but because this test doesn't require that level of detail, I guess it wouldn't be efficient to implement this approach. Thanks for the insight!
